integration/integrate-with-ci-systems.adoc

```diff
@@ -7,34 +7,41 @@ include::modules/common-attributes.adoc[]
+{product-title} ({product-title-short}) integrates with a variety of continuous integration (CI) products and allows you to apply build-time and deploy-time security rules before you deploy images.
 
-{product-title} integrates into CI pipelines after images are built and pushed to a registry.
+{product-title-short} integrates into CI pipelines after images are built and pushed to a registry.
 Pushing the image first allows developers to continue testing their artifacts while dealing with any policy violations alongside any other CI test failures, linter violations, or other problems.
 
-If possible, you should configure the version control system to block pull or merge requests from being merged if the build stage, which includes {product-title} checks, fails.
+If possible, you should configure the version control system to block pull or merge requests from being merged if the build stage, which includes {product-title-short} checks, fails.
 
-The integration with your CI product functions by contacting your {product-title} installation to check whether the image complies with build-phase policies you have configured.
+The integration with your CI product functions by contacting your {product-title-short} installation to check whether the image complies with build-time policies you have configured.
 If there are policy violations, a detailed message is displayed on the console log, including the policy description, rationale, and remediation instructions.
 Each policy includes an optional enforcement setting;
-if you mark a policy for build-phase enforcement, failure of that policy causes the client to exit with a nonzero error code.
+if you mark a policy for build-time enforcement, failure of that policy causes the client to exit with a nonzero error code.
 
 To integrate {product-title} with your CI system, follow these steps:
 
 . xref:../integration/integrate-with-ci-systems.adoc#configure-build-policies[Configure build policies].
 . xref:../integration/integrate-with-ci-systems.adoc#configure-registry-integration[Configure a registry integration].
-. xref:../integration/integrate-with-ci-systems.adoc#configure-access[Configure access] to your {product-title} instance.
+. xref:../integration/integrate-with-ci-systems.adoc#configure-access[Configure access] to your {product-title-short} instance.
 . xref:../integration/integrate-with-ci-systems.adoc#integrate-with-your-ci-pipeline[Integrate with your CI pipeline].
 
 [id="configure-build-policies"]
 == Configuring build policies
 
-To check {product-title} policies during builds, you must first configure policies that apply to the build phase of the container lifecycle. And then you must integrate with the registry that images are pushed to during the build.
+To check {product-title-short} policies during builds, you must first configure policies that apply to the build time of the container lifecycle. Then, you must integrate with the registry that images are pushed to during the build.
```

Comment:
Do we an example/steps to configure a build time policy in our documentation?

Reply:
I need to check on your first question. We talk about generating build-time network policies with the roxctl netpol generate command, but I don't think that is what you are asking about here. If we need to come up with an example, I will create a ticket for this and do it in a separate PR.

For your second question, I added a link to the registry integration chapter right after the section on "configuring build policies."

Reply:
Ticket to create a config example: https://issues.redhat.com/browse/ROX-23585